python/ray/tune/trial_runner.py

```diff
@@ -688,7 +690,10 @@ def _update_trial_queue(self, blocking=False, timeout=600):
     def request_stop_trial(self, trial):
         self._stop_queue.append(trial)
 
-    def _process_requests(self):
+    def request_stop_experiment(self):
```

Comment:
nit: documentation explaining behavior (eg when the request will actually be processed)